Library/Homebrew/build.rb

```diff
@@ -83,7 +83,8 @@ def install
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
       ENV.run_time_deps = run_time_deps
-      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
+      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) } ||
+                deps.any? { |dep| dep.name == "linuxbrew/xorg/xorg" }
```

Comment:
In general, I'm OK with this. It makes sense and is easy to read. I'll note that I would probably write it as an additional `ENV`:
```
ENV.xorg = deps.any? { |dep| dep.name == "linuxbrew/xorg/xorg" }
```
and then something like
```
alias x11? x11 or xorg
```
...
but it would probably be more work. not sure.